tests/e2e/cucumber/steps/ui/links.ts

```diff
@@ -231,3 +231,18 @@ When(
     await pageObject.openShareWithMeFromInternalLink(actor)
   }
 )
+
+When(
+  '{string} copies the link named {string} to clipboard of resource {string}',
```

Comment:
maybe this is enough?
```suggestion
  '{string} copies the link {string} of resource {string}',
```